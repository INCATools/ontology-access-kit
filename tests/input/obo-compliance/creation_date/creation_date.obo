format-version: 1.4
ontology: creation_date

[Term]
id: X:1
creation_date: 2023-11-20

