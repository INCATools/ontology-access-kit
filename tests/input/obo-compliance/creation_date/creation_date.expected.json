{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/creation_date.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "type" : "CLASS",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://www.geneontology.org/formats/oboInOwl#creation_date",
          "val" : "2023-11-20"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#creation_date",
      "lbl" : "creation date",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ]
  } ]
}